# custom.msg
float64 robpos_x
float64 robpos_y
float64 robvel_x
float64 robvel_y